pkg/scheduler/profile/profile.go

```diff
@@ -36,33 +36,22 @@ type RecorderFactory func(string) events.EventRecorder
 // FrameworkFactory builds a Framework for a given profile configuration.
 type FrameworkFactory func(config.KubeSchedulerProfile, ...frameworkruntime.Option) (framework.Framework, error)
```

Comment:
I don't quite like this factory after Profile is removed. Could you, in a follow up PR, work on removing it and calling runtime.NewFramework directly?

Reply:
> I don't quite like this factory after Profile is removed. Could you, in a follow up PR, work on removing it and calling runtime.NewFramework directly?

I agree with you.